Dask Engine Testing.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
@chukarsten please take this notebook off your branch before merging (or make it a docs demo!)

Reply:
yup, I need to update it for use with this branch as I think its a design iteration behind, but just wanted to include it for convenience for now in case anyone wanted to mess with it